ui-components/src/page_components/listing/listing_sidebar/ReferralApplication.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react"
+import { t } from "../../../helpers/translator"
+import { Icon } from "../../../icons/Icon"
+
+interface ReferralApplicationProps {
+  description: string
+  phoneNumber: string
+  title: string
+}
+
+const ReferralApplication = (props: ReferralApplicationProps) => {
+  const linkedPhoneNumber = `tel:${props.phoneNumber.replace(/[-()]/g, "")}`
+
+  return (
+    <section className="aside-block -mx-4 pt-0 md:mx-0 md:pt-4">
```

Comment:
We can remove all the utility classes now and just use `aside-block` (due to merge of #1140)